opm/output/eclipse/AggregateGroupData.hpp

```diff
@@ -124,14 +124,17 @@ class AggregateGroupData
                                                            {"FWPR",  1},
                                                            {"FGPR",  2},
                                                            {"FVPR",  3},
+                                                           {"FOIR",  4},
                                                            {"FWIR",  5},
                                                            {"FGIR",  6},
+                                                           {"FVIR",  7},
                                                            {"FWCT",  8},
                                                            {"FGOR",  9},
                                                            {"FOPT", 10},
                                                            {"FWPT", 11},
                                                            {"FGPT", 12},
                                                            {"FVPT", 13},
+                                                           {"FOIT", 14},
```

Comment:
This will end up outputting the injection rates to the `XGRP` restart vector.  Did you verify that the item IDs (e.g., `FOIR = 4`) are the actual locations where we should write those rates/cumulatives?